WorkflowFunctionBundle/Tests/Functional/Repository/WorkflowFunctionRepositoryTest.php

```diff
@@ -2,6 +2,8 @@
 
 namespace OpenOrchestra\WorkflowFunctionBundle\Tests\Functional\Repository;
 
+use OpenOrchestra\ModelInterface\Repository\Configuration\FinderConfiguration;
+use OpenOrchestra\ModelInterface\Repository\Configuration\PaginateFinderConfiguration;
```

Comment:
i think there is an error in composer.json of workflowFunction ModelInterfaceBundle isn't in require. the bundle works because ModelInterface is require by cms-bundle, so i tkink should be add  ModelInterfaceBundle in require.